python/pyspark/tests/test_worker.py

```diff
@@ -191,8 +191,14 @@ def test_reuse_worker_of_parallelize_range(self):
         rdd = self.sc.parallelize(range(20), 8)
```

Comment:
@Yikun, maybe just run the eventually with the whole test here. e.g.)

```python
def test_reuse_worker_of_parallelize_range(self):
    def check_reuse_worker_of_parallelize_range():
        rdd = self.sc.parallelize(range(20), 8)
        previous_pids = rdd.map(lambda x: os.getpid()).collect()
        current_pids = rdd.map(lambda x: os.getpid()).collect()

        for pid in current_pids:
            self.assertTrue(pid in previous_pids)

    eventually(check_reuse_worker_of_parallelize_range, catch_assertions=True)
```

Reply:
Ah, yes for sure